examples/kms/vault/user-secret.yaml

```diff
@@ -0,0 +1,11 @@
+---
+# This is the user secret containing encryptionPasspharse that can be
+# created in a Kubernetes Namespace for encrypting PVCs with the
+# "user-ns-secrets-metadata-test" or "user-secrets-metadata-test"
```

Comment:
`user-ns-secrets-metadata-test" or "user-secrets-metadata-test` are encryptionID not encryptionKMSType isnt it?

Reply:
yes, I'll change it :sweat_smile: